modules/configuration/ajax/process.php

```diff
@@ -27,39 +27,91 @@
 $client = new NDB_Client();
 $client->makeCommandLine();
 $client->initialize();
-
 $DB =& Database::singleton();
 foreach ($_POST as $key => $value) {
     if (is_numeric($key)) { //update
         if ($value == "") {
             $DB->delete('Config', array('ID' => $key));
         } else {
-            $DB->update(
-                'Config',
-                array('Value' => $value),
-                array('ID' => $key)
-            );
+            // if no duplicate value then do updating
+            if (duplicateExistsInDropdown($key, $value)) {
+                $DB->update(
+                    'Config',
+                    array('Value' => $value),
+                    array('ID' => $key)
+                );
+            } else {
+                   header("HTTP/1.1 303 Duplicate value for update");
```

Comment:
This isn't a valid HTTP response code. 303 is 'See Other', intended for redirects. 

I think it should just be a 400 Bad Request (or maybe a 409 Conflict) response.

Reply:
Thanks, done.